backend-shared/config/db/application_test.go

```diff
@@ -0,0 +1,191 @@
+package db
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetApplicationById(t *testing.T) {
+	testSetup(t)
+	defer testTeardown(t)
+	dbq, err := NewUnsafePostgresDBQueries(true, true)
+	if !assert.NoError(t, err) {
+		return
+	}
+	defer dbq.CloseDatabase()
+
+	ctx := context.Background()
+
+	_, managedEnvironment, _, gitopsEngineInstance, _, err := createSampleData(t, dbq)
+	if !assert.NoError(t, err) {
+		return
+	}
+	applicationput := Application{
+		Application_id:          "test-my-application-1",
+		Name:                    "test-application",
+		Spec_field:              "{}",
```

Comment:
Remove unwanted details if not necessary

Reply:
Hi Samyak thanks for your review. I tried pruning at all the instances you pointed out. But all these specifications are crucial, when I try to test by removing them I get the following error 
`Error Trace:	application_test.go:33`

        	Error:      	Received unexpected error:

        	            	Spec_field field should not be empty string, in CreateApplication

        	Test:       	TestGetApplicationById

Applies on all the comments regarding removal of unwanted details as of commit 0459875